### PR TITLE
Refine CODEOWNERS to include Infrastructure Reviewers

### PR DESCRIPTION
**Pull request by giorgiosironi**
During https://github.com/libero/publisher/issues/310 [Node.js Reviewers](https://github.com/orgs/libero/teams/node-js-reviewers) were continuously requested for on scripts that aren't really related.

### CODE REVIEW

**Comment by giorgiosironi** [comment]
Though if Node.js reviewers are required anyway to merge, would need to request both?

**Comment by thewilkybarkid** [comment]
Think it's ok. Could also have a merger group, but there's enough cross over anyway.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1,3 @@
 * @libero/node-js-reviewers
+.github/* @libero/infrastructure-reviewers
+.docker/* @libero/infrastructure-reviewers
```
Comment (thewilkybarkid) [inline .github/CODEOWNERS]: Might want to include the `Dockerfile` and `Makefile`?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1 +1,5 @@
 * @libero/node-js-reviewers
+.docker/* @libero/infrastructure-reviewers
+.github/* @libero/infrastructure-reviewers
+Dockerfile @libero/infrastructure-reviewers
+Makefile @libero/infrastructure-reviewers
```
Comment (thewilkybarkid) [inline .github/CODEOWNERS]: Reading https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax these should be

```
/.docker/ @libero/infrastructure-reviewers
/.github/ @libero/infrastructure-reviewers
/Dockerfile @libero/infrastructure-reviewers
/Makefile @libero/infrastructure-reviewers
```

Also want `/.dockerignore`. `/.scripts` too? (Does that count as a file here? 🤔)

Not sure about `/.dependabot/`...

Comment (giorgiosironi) [reply .github/CODEOWNERS]: In theory `.scripts` should be a file containing the submodule commit so does count for modifications.

I'll go out on a limb and say `/.dependabot` is developer-oriented like the linting or testing configuration so shouldn't be included. It isn't very language-specific still.

**Review by thewilkybarkid — APPROVED** [review]

**Pull request merged**